packages/now-cli/package.json

```diff
@@ -61,6 +61,15 @@
   "engines": {
     "node": ">= 10"
   },
+  "dependencies": {
+    "@now/build-utils": "2.2.2-canary.1",
+    "@now/go": "1.0.7",
+    "@now/next": "2.5.5-canary.0",
+    "@now/node": "1.5.2-canary.2",
+    "@now/python": "1.1.6",
+    "@now/ruby": "1.1.0",
```

Comment:
Any reason why these are not using canary versions here?

Reply:
They're using the versions specified in the respective local source `package.json` files. So for example, the current version in the `@now/ruby` tree happens to be a stable release, not a canary.

I was discussing with @leo on the call earlier about how we'll need to update the publish logic to update these versions according to whether or not it's a stable release. That part's still TODO.

Reply:
Oh I see. I realize that we just published canary `@now/python` so that threw me off.

I don't think there is anything special we need to do here, lerna should take care of the versioning so that bumping the Runtime version will also bump it here (we used to do the same with `@now/node-bridge` because it was a dependency).